members/CusR53252e1d1510267638.yaml

```diff
@@ -0,0 +1,113 @@
+bioguide: CusTef250d981600120189
```

Comment:
ID needs to match file name

Reply:
AHH, good call, that was def a copy/pasta mistake from Victor's old YAML. 🔔